.github/workflows/main.yml

```diff
@@ -4,15 +4,13 @@ on:
     branches:
     - master
     - "3.0"
-    - "2.19"
```

Comment:
Unfortunately, I think master branch is used for all GH builds. So this could stop 2.19 builds. Worth testing in one repo before merging similar changes to all repos.

Reply:
Or it could be the default branch that is used to control all GH builds and not master. 2.19 is defined as the default branch in this repo.